README.md

```diff
@@ -70,23 +70,25 @@ Sourcebot supports indexing and searching through public and private repositorie
     cd sourcebot_workspace
     ```
 
-2. Create a new config following the [configuration schema](./schemas/index.json) to specify which repositories Sourcebot should index. For example, to index [llama.cpp](https://github.com/ggerganov/llama.cpp):
+2. Create a new config following the [configuration schema](./schemas/v2/index.json) to specify which repositories Sourcebot should index. For example, let's index llama.cpp:
 
     ```sh
     touch my_config.json
     echo '{
-        "$schema": "https://raw.githubusercontent.com/sourcebot-dev/sourcebot/main/schemas/index.json",
-        "Configs": [
+        "$schema": "https://raw.githubusercontent.com/sourcebot-dev/sourcebot/refs/tags/latest/schemas/v2/index.json",
+        "repos": [
             {
-                "Type": "github",
-                "GitHubUser": "ggerganov",
-                "Name": "^llama\\\\.cpp$"
+                "type": "github",
+                "repos": [
+                    "ggerganov/llama.cpp"
+                ]
             }
         ]
     }' > my_config.json
     ```
 
-    (For more examples, see [example-config.json](./example-config.json). For additional usage information, see the [configuration schema](./schemas/index.json)).
+    >[!NOTE] 
```

Comment:
nit: these NOTE highlights seem to be broken?

Reply:
ah kk - I think GitHub doesn't like it when you indent these. Will change.